tests/models/test_models_unet_2d_condition.py

```diff
@@ -270,6 +270,12 @@ def prepare_init_args_and_inputs_for_common(self):
         inputs_dict = self.dummy_input
         return init_dict, inputs_dict
 
+    def test_memory_footprint(self):
```

Comment:
Can you also load the model in fp16 here and verify that it's half the size then?